### ISSUE TITLE
Compilation with -DCHECK fails in src_c/IMB.c

### ISSUE THREAD

**Issue by kawashima-fj** [intel/mpi-benchmarks#9]
`make -C src_c CFLAGS=-DCHECK` fails because of missing `(` in `IMB.c`.

```patch
diff --git src_c/IMB.c src_c/IMB.c
index 21ded36..ae97b86 100644
--- src_c/IMB.c
+++ src_c/IMB.c
@@ -388,7 +388,7 @@ Return value          (type int)
     IMB_free_all(&C_INFO, &BList, &ITERATIONS);
 
 #ifdef CHECK
-    if num_alloc == num_free)
+    if (num_alloc == num_free)
         ierr=0;
     else {
         fprintf(stderr, "pr %d: calls to IMB_v_alloc %d / IMB_v_free %d (doesn't seem ok, are unequal!)\n", C_INFO.w_rank,num_alloc,num_free);
```

**Comment by VinnitskiV** [comment]
@kawashima-fj 
Thank you for the contribution. This issue will be fixed in one of upcoming releases, with corresponding notes in Release Notes (What’s New section). As soon the release is out, this issue will be closed.

**Comment by kawashima-fj** [comment]
I confirmed this is fixed in IMB 2019.2. Thanks.
